tensorrt_llm/llmapi/llm_args.py

```diff
@@ -1446,6 +1441,89 @@ def model_post_init(self, __context):
         if self.parallel_config.auto_parallel:
             self.parallel_config.world_size = self.auto_parallel_world_size
 
+        # Setup build config after model initialization
+        self._setup_build_config()
+        self._setup_enable_build_cache()
+        self._setup_speculative_config()
+        self._setup_embedding_parallel_mode()
+
+        self.calib_config = self.calib_config or CalibConfig()
```

Comment:
Could be handled with `Field(default_factory=CalibConfig, ...)`

Reply:
Good catch, updated.